client/python/tensorflow-serving-api/samples/README.md

```diff
@@ -370,7 +373,7 @@ python rest_get_model_metadata.py --rest_port 8000
    "signatureDef": {
     "serving_default": {
      "inputs": {
-      "data": {
+      "0": {
```

Comment:
also to update:
"name": "data" -> "name": "0" (line 399)

Reply:
fixed